### DISCUSSION TITLE
Error when try to connect api from postman

### DISCUSSION THREAD

**Discussion by rahooftkh** [serenity-is/Serenity#6336] [Help]
hi
iam try to access my api from postman post request but get an error 400 bad request

**Comment by edwardch** [comment]
I think if you provide more information you might get a better chance of a good answer.

What url, parameters, app version, etc.

**Comment by minhhungit** [comment]
code 400  mean your url is incorrect

**Comment by wezmag** [comment]
I had the same issue this morning.
I fixed it by adding `[IgnoreAntiforgeryToken]` attribute on the API controller or action.
However, you will need to implement some token based authentication to protect you API endpoints.
